docs/advanced-features/athena.md

```diff
@@ -1,91 +1,475 @@
+                "s3:ListBucket",
+                "s3:ListBucketMultipartUploads",
+                "s3:ListMultipartUploadParts",
+            ],
+            "Resource": [
+                "*",
+            ],
+        },
+        {
+            "Sid": "GrantQuiltAthenaInputAccess",
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetObject",
+            ],
+            "Resource": [
+                "arn:aws:s3:::*",
+                "arn:aws:s3:::*/.quilt/*",
+            ],
+        },
+        {
+            "Sid": "GrantQuiltAthenaOutputAccess",
+            "Effect": "Allow",
+            "Action": [
+                "s3:GetBucketLocation",
+                "s3:GetObject",
+                "s3:ListBucket",
+                "s3:ListBucketMultipartUploads",
+                "s3:ListMultipartUploadParts",
+                "s3:AbortMultipartUpload",
+                "s3:CreateBucket",
+                "s3:PutObject",
+                "s3:PutBucketPublicAccessBlock",
+            ],
+            "Resource": [
+                ARN_ATHENA,
+                ARN_ATHENA + "/*",
+            ],
+        },
+    ],
+}
+#print(AthenaQuiltAccess)
+```
+
+    {'Version': '2012-10-17', 'Statement': [{'Sid': 'GrantQuiltAthenaFullAccess', 'Effect': 'Allow', 'Action': ['athena:ListWorkGroups'], 'Resource': ['*']}, {'Sid': 'GrantQuiltAthenaAccess', 'Effect': 'Allow', 'Action': ['athena:Create*', 'athena:Get*', 'athena:ListDatabases', 'athena:ListNamedQueries', 'athena:ListPreparedStatements', 'athena:ListQueryExecutions', 'athena:StartQueryExecution'], 'Resource': ['arn:aws:glue:us-east-1:712023778557:catalog', 'arn:aws:glue:us-east-1:712023778557:datacatalog/*', 'arn:aws:athena:us-east-1:712023778557:workgroup/quilt-query']}, {'Sid': 'GrantQuiltGlueAccess', 'Effect': 'Allow', 'Action': ['glue:GetDatabase', 'glue:GetDatabases', 'glue:GetTable', 'glue:GetTables'], 'Resource': ['*']}, {'Sid': 'GrantQuiltGlueWriteAccess', 'Effect': 'Allow', 'Action': ['glue:CreateTable', 'glue:DeleteTable', 'glue:UpdateTable'], 'Resource': ['arn:aws:glue:us-east-1:712023778557:catalog', 'arn:aws:glue:us-east-1:712023778557:database/quilt_query', 'arn:aws:glue:us-east-1:712023778557:table/quilt_query/*_quilt_*']}, {'Sid': 'GrantQuiltAthenaBucketAccess', 'Effect': 'Allow', 'Action': ['s3:GetBucketLocation', 's3:ListBucket', 's3:ListBucketMultipartUploads', 's3:ListMultipartUploadParts'], 'Resource': ['*']}, {'Sid': 'GrantQuiltAthenaInputAccess', 'Effect': 'Allow', 'Action': ['s3:GetObject'], 'Resource': ['arn:aws:s3:::*', 'arn:aws:s3:::*/.quilt/*']}, {'Sid': 'GrantQuiltAthenaOutputAccess', 'Effect': 'Allow', 'Action': ['s3:GetBucketLocation', 's3:GetObject', 's3:ListBucket', 's3:ListBucketMultipartUploads', 's3:ListMultipartUploadParts', 's3:AbortMultipartUpload', 's3:CreateBucket', 's3:PutObject', 's3:PutBucketPublicAccessBlock'], 'Resource': ['arn:aws:s3:::mycompany-quilt-query-results', 'arn:aws:s3:::mycompany-quilt-query-results/*']}]}
```

Comment:
Do we really want it looking like this?
![image](https://user-images.githubusercontent.com/481910/186110956-552cf950-55f3-4850-adf6-a75500d992f6.png)

Reply:
Leftover from prior run. Removed.